app/views/course/assessment/answer/programming_file_annotations/_discussion_topic_programming_file_annotation.html.slim

```diff
@@ -1,17 +1,22 @@
 - file_annotation = discussion_topic_programming_file_annotation
+- topic = file_annotation.acting_as
 - answer = file_annotation.file.answer
 - question = answer.question
 - submission = answer.submission
 - assessment = question.assessment
 
-h3
-  = link_to assessment.title, edit_course_assessment_submission_path(current_course, assessment, submission)
-h4
-  = t('.by_html', user: link_to_user(submission.creator))
-h4
-  = t('.question', title: question.title)
+= div_for(topic) do
+  h3
+    = link_to assessment.title, edit_course_assessment_submission_path(current_course, assessment, submission)
+  h4
+    = t('.by_html', user: link_to_user(submission.creator))
+  h4
+    = t('.question', title: question.title)
 
-= display_code_lines(file_annotation.file, file_annotation.line - 5, file_annotation.line)
+  = display_code_lines(file_annotation.file, file_annotation.line - 5, file_annotation.line)
 
-= render partial: 'course/discussion/topic', object: file_annotation.acting_as,
-         locals: { post_partial: 'course/assessment/answer/programming/annotation_post' }
+  = render partial: 'course/discussion/topic', object: topic,
```

Comment:
maybe this should be taken out as a helper? I remember repeating this in some places